src/styles/drawer-content/drawer-content.ts

```diff
@@ -0,0 +1,54 @@
+import { makeStyles } from "@material-ui/core";
+import theme from "theme/theme";
+
+export const useDrawerContentStyles = makeStyles({
+
+  drawerContentContainer: {
+    display: "flex",
+    alignItems: "center",
+    margin: `${theme.spacing(2)}px 0px`,
+  },
+
+  searchBoxContaienr: {
+    height: 40,
+    flexGrow: 200,
+    position: "relative",
+  },
+
+  searchIcon: {
+    top: "50%",
+    color: "rgba(0, 0, 0, 0.8)",
+    left: theme.spacing(1),
+    position: "absolute",
+    transform: "translateY(-50%)",
+  },
+
+  searchBox: {
+    height: "100%",
+    width: "100%",
+    "& div div": {
```

Comment:
Is there a reason why these cannot be their own classes that those selected elements can use directly?